contents/handbook/people/team-structure/app-small-teams.md

```diff
@@ -0,0 +1,41 @@
+---
+title: App Small Teams
+sidebar: Handbook
+showTitle: true
+hideAnchor: false
+---
+
+App Small Teams are particular types of Small Teams that own an area of the product.
+
+## What does owning an area of the product mean?
```

Comment:
Also makes me want to ask "what are the areas of the product?"

Reply:
not sure I understand, would you mind rephrasing?

Reply:
I suppose I'm asking... do we have a list of the products/product teams?

* session recordings
* product analytics
* flags and experimentation
* ingestion
* platform

??

Is every team a product team?